### ISSUE TITLE
train script missing

### ISSUE THREAD

**Issue by sheshap** [hehefan/PSTNet2#1]
Dear @hehefan ,

Please upload the training scripts (as it contains all the training parameters along with network design parameters).
Thanks in advance

**Comment by hehefan** [comment]
Hi,

This project is an extension to [PSTNet](https://github.com/hehefan/Point-Spatio-Temporal-Convolution). Please refer to PSTNet for the training scripts.

Best regards.

**Comment by sheshap** [comment]
Hi,

I have used that and got only 87.45% accuracy.
Can you please suggest how to enable Spatio-temporal hierarchy (hier) and Spatio-temporal decomposition (decom)?

Thanks and Regards,
Shivanand

**Comment by hehefan** [comment]
Hi,

Hierarchy and decomposition are default.  Could you please check whether the environment is aligned with the proposed? 

Best.

**Comment by sheshap** [comment]
Hi,

I am on a workstation Ubuntu 20.04, CUDA 11.6, GCC 9.4, Torch 1.11.0 (successfully compiled code in modules 1.8 folders), and cuDNN 8.

However, I used the same environment and was able to recreate the results of the P4Transformer and PSTNet.

Thanks

**Comment by hehefan** [comment]
Hi,

I will try to set up the same environment as you provided and reproduce the problem. 

Best.

**Comment by sheshap** [comment]
Hi,

Please let us know if you are able to reproduce the problem.

Thanks

**Comment by sheshap** [comment]
Hi,

I got 90.59% on PSTNet2 with clip-len = 24.
Still less than the reported accuracy in the paper.
Please help us recreate the results.

Thanks

**Comment by hehefan** [comment]
Hi,

Could you please align the environment according as follows?  

```
Red Hat Enterprise Linux Workstation release 7.7 (Maipo), g++ (GCC) 8.3.1, PyTorch v1.4.0, CUDA 10.2 and cuDNN v7.6.
```

As we discussed many times before, because MSRAction-3D is very small, the accuracy may be influenced by random sampling. 

My experiment logs are in [PSTNet++.tar.gz](https://drive.google.com/file/d/1_vmIsfSW6cc5-Wc6QNP3ucvz7VH7bzFT/view?usp=sharing). You may use them as a reference. 


Best regard.
